src/rpc/introduction.md

```diff
@@ -0,0 +1,39 @@
+# Remote Procedure Calls (RPC)
+
+Communication between services occurs over remote procedure calls. RPCs happen all the time in distributed systesm. You browser had to make at least one RPC to this website to obtain this webpage.

```

Comment:
```suggestion
Communication between processes can use remote procedure calls. RPCs happen all the time in distributed systems. You browser had to make at least one RPC to this website to obtain this webpage.
```

I'm not sure if HTTP is a kind of RPC, I don't think I would classify it as such. For me, RPC implies a procedure-like interface, rather than a networking protocol style interface.